cfme/tests/automate/custom_button/test_buttons.py

```diff
@@ -547,3 +547,61 @@ def test_attribute_override():
         1651099
     """
     pass
+
+
+@pytest.mark.meta(blockers=[BZ(1719282, unblock=lambda button_type: button_type != "User")])
+@pytest.mark.parametrize("button_type", ["User", "Provider"])
+@pytest.mark.provider([VMwareProvider], override=True, scope="module", selector=ONE_PER_TYPE)
+def test_simulated_object_copy_on_button(appliance, provider, setup_provider, button_type):
+    """ Test copy of simulated object over custom button
+
+    Polarion:
+        assignee: ndhandre
+        initialEstimate: 1/4h
+        caseimportance: medium
+        caseposneg: positive
+        casecomponent: CustomButton
+        tags: custom_button
+        testSteps:
+            1. simulate button with Automate -> Simulation
+            2. copy simulated data
+            3. paste simulated data on button from Automate -> Customizationn -> Buttons
+            4. check copy-paste working or not
+
+    Bugzilla:
+        1426390
+        1719282
+    """
+    if button_type == "User":
+        target_type = "{}User".format("EVM " if appliance.version < "5.11" else "")
+        target_obj = "Administrator"
+    else:
+        target_type = "Provider"
+        target_obj = provider.name
+
+    # simulate and copy
+    view = navigate_to(appliance.server, "AutomateSimulation")
+
+    simulation_data = {
+        "instance": "Automation",
+        "message": "test_bz",
+        "request": "InspectMe",
+        "target_type": target_type,
+        "target_object": target_obj,
+        "execute_methods": True,
+    }
```

Comment:
Is there any need for making a `Collection` for an `AutomateSimulation`? As far as I can tell, they are not really permanent objects, just temporary. But it may simplify the few test cases that use them. Do you think it's worth it?

Reply:
I thought same but I think we have one method.

Reply:
As it's not Entity based just one operation l.e. `simulate` so no need to implement collection for now. I improved some arguments like `appliance` needs for navigation.